docs/reference/deprecated.md

```diff
@@ -1,5 +1,9 @@
 Gutenberg's deprecation policy is intended to support backwards-compatibility for releases, when possible. The current deprecations are listed below and are grouped by _the version at which they will be removed completely_. If your plugin depends on these behaviors, you must update to the recommended alternative before the noted version.
 
+## 5.1
```

Comment:
I'm unsure how many plugin versions we're going to release before WordPress 5.0 is out. We are also in API-freeze mode, so I guess in this case makes sense to remove the component in the second planned version after public release.

Reply:
I think Gutenberg 4.3 and 4.4 will ship and 4.4 will be what merges into WordPress 5.0.

Reply:
So this should not be 5.1; those are WordPress version numbers. These deprecations are relevant to Gutenberg versions, where are separate.

Reply:
```suggestion
## 4.6
```